pkg/registry/apiserverinternal/storageversion/strategy.go

```diff
@@ -96,8 +103,71 @@ func (storageVersionStatusStrategy) PrepareForUpdate(ctx context.Context, obj, o
 
 	newSV.Spec = oldSV.Spec
 	metav1.ResetObjectMetaForStatus(&newSV.ObjectMeta, &oldSV.ObjectMeta)
+
+	status := apiserverinternal.ConditionFalse
```

Comment:
I don't understand why we would set this condition in the server here. Can we split the condition into another PR?

Reply:
will do it in a separate PR